EIPS/eip-4762.md

```diff
@@ -89,25 +89,50 @@ Where `tree_key` and `sub_key` are computed as `tree_key, sub_key = get_storage_
-     * The destination of a `JUMP` (or positively evaluated `JUMPI`) is considered to be accessed, even if the destination is not a jumpdest or is inside pushdata
-     * The destination of a `JUMPI` is not considered to be accessed if the jump conditional is `false`.
-     * The destination of a jump is not considered to be accessed if the execution gets to the jump opcode but does not have enough gas to pay for the gas cost of executing the `JUMP` opcode (including chunk access cost if the `JUMP` is the first opcode in a not-yet-accessed chunk)
-     * The destination of a jump is not considered to be accessed if it is beyond the code (`destination >= len(code)`)
-     * If code stops execution by walking past the end of the code, `PC = len(code)` is not considered to be accessed
- * If the current step of EVM execution is a `PUSH{n}`, all chunks `(PC // CHUNK_SIZE) <= chunk_index <= ((PC + n) // CHUNK_SIZE)` of the callee are accessed.
- * If a nonzero-read-size `CODECOPY` or `EXTCODECOPY` read bytes `x...y` inclusive, all chunks `(x // CHUNK_SIZE) <= chunk_index <= (min(y, code_size - 1) // CHUNK_SIZE)` of the accessed contract are accessed.
-     * Example 1: for a `CODECOPY` with start position 100, read size 50, `code_size = 200`, `x = 100` and `y = 149`
-     * Example 2: for a `CODECOPY` with start position 600, read size 0, no chunks are accessed
-     * Example 3: for a `CODECOPY` with start position 1500, read size 2000, `code_size = 3100`, `x = 1500` and `y = 3099`
- * `CODESIZE`, `EXTCODESIZE` and `EXTCODEHASH` do NOT access any chunks.
-    When a contract is created, access chunks `0 ... (len(code)+30)//31`
+When a contract is created, access all corresponding chunks `0 ... (cod_size+30)//31`, where `code_size` is `len(code)` in legacy contracts, or the container size in EOFv1 accounts.
+
+#### Access events for EOFv1 container
+
+If a `*CALL`, `EXT*CALL` instruction targets an account with the EOFv1 container format or a `EOFCREATE` instruction is executed, the code chunks of the following bytecodes are considered accessed:
+
+* The entire `header` section of the current (`*CALL`/`EXT*CALL`) or targeted (`EOFCREATE`) (sub-)container.
+* The entire `type_section[0]` of the current or targeted (sub-)container.
+
+If a `CALLF` or `JUMPF` instruction is executed, the corresponding code chunks of the target's entire `types_section`  and the first byte of the `code_section` for the target section are considered accessed.
+
+If a `RETURNCONTRACT` instruction is executed, the corresponding code chunks of the target's entire `types_section` and `code_section` for the target section are considered accessed.
+
+If `DATALOAD`, `DATALOADN` or `DATACOPY` is executed, the corresponding code chunks of the `data_section` of the current container are considered accessed. If `DATALOAD` or `DATACOPY` access bytes outside the `data_section`, those are not accessed (i.e: are assumed to be zero by [EIP-7480](./eip-7480.md)).
```

Comment:
The last sentence might be redundant but preferred to clarify. This doesn't happen with `DATALOADN` since this is validated at creation time.